src/sentry/snuba/metrics/extraction.py

```diff
@@ -174,6 +178,10 @@
 
 Variables = Dict[str, Any]
 
+unresolvedQueryBuilder = UnresolvedQuery(
```

Comment:
I really hope we can remove the dependency on any `*QueryBuilder` code in the future. I would dream of having a central place to handle parsing that supports multiple of our internal grammars and that we could easily reuse without initializing a lot of unnecessary code.

```suggestion
query_builder = UnresolvedQuery(
```